### PR TITLE
Task4618 - Vuetify Switches documentation

### PR DESCRIPTION
**Pull request by annav00**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -732,45 +732,56 @@ For examples of usage see: [JDI Vuetify Sliders tests](https://github.com/jdi-te
...
+**isReadonly()** | Checks that switch is readonly                  | boolean
+**hasMessages()** | Check that switch has messages                  | boolean
+**hasErrorMessage()** | Check that switch has error messages            | boolean
+**hasSuccessMessage()** | Check that switch has success messages          | boolean
+**hasIconAppend()** | Check that switch has icon-append               | boolean
+**hasIconPrepend()** | Check that switch has icon-prepend              | boolean
+**hasRipple()** | Check that switch has ripple                    | boolean
+**is()** | Switch assert                                   | SwitchAssert
+
+For examples of usage see: [JDI Vuetify Switch tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SwitchTests.java).
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:785]: Hi :) failed to find this point "7.	Element code (with reference to framework version - IMPORTANT!)" please, clarify

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -732,45 +732,78 @@ For examples of usage see: [JDI Vuetify Sliders tests](https://github.com/jdi-te
...
+        <div class="v-input--switch__track theme--light">
+        </div>
+        <div class="v-input--switch__thumb theme--light">
+          <!---->
+        </div>
+      </div>
+      <label for="input-1187" class="v-label theme--light" style="left: 0px; right: auto; position: relative;">red</label>
+    </div>
+  </div>
+</div>
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:778]: Sorry, my bad, did not notice at first. But it is not Vuetify code.
Shouldn't we have example of Vuetify template like this one?
It's a question :) Maybe you agreed with smb that it shoud be html?
`<template>
  <v-container
    class="px-0"
    fluid
  >
    <v-switch
      v-model="switch1"
      :label="`Switch 1: ${switch1.toString()}`"
    ></v-switch>
  </v-container>
</template>`

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:778]: Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 
image

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (annav00) [reply source/includes/documentation/_vue_js_elements.md:782]: Added a comment with FinedBy

File: source/includes/documentation/_vue_js_elements.md [hunk 3]
```diff
@@ -732,45 +732,82 @@ For examples of usage see: [JDI Vuetify Sliders tests](https://github.com/jdi-te
...
+  redSwitch.show();
+  redSwitch.is().checked();
+  redSwitch.label().has().text(red);
+  redSwitch.has().value(red);
+  redSwitch.has().color(RED.value());
+  redSwitch.has().detailsHidden();
+  redSwitch.uncheck();
+  redSwitch.is().unchecked();
+}
 ```
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:758]: please, update test :) now it has description :)

File: source/includes/documentation/_vue_js_elements.md [hunk 4]
```diff
@@ -732,45 +732,82 @@ For examples of usage see: [JDI Vuetify Sliders tests](https://github.com/jdi-te
...
+**isReadonly()** | Checks that switch is readonly                  | boolean
+**hasMessages()** | Check that switch has messages                  | boolean
+**hasErrorMessage()** | Check that switch has error messages            | boolean
+**hasSuccessMessage()** | Check that switch has success messages          | boolean
+**hasIconAppend()** | Check that switch has icon-append               | boolean
+**hasIconPrepend()** | Check that switch has icon-prepend              | boolean
+**hasRipple()** | Check that switch has ripple                    | boolean
+**is()** | Switch assert                                   | SwitchAssert
+
+For examples of usage see: [JDI Vuetify Switch tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SwitchTests.java).
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:810]: the link does not work :(

**Review by MayaElf — CHANGES_REQUESTED** [review]

**Review by MayaElf — APPROVED** [review]

**Review by vklonin — APPROVED** [review]

**Pull request merged**